packages/assets-controllers/src/NftDetectionController.ts

```diff
@@ -56,7 +56,10 @@ export type NftDetectionControllerMessenger = RestrictedControllerMessenger<
   AllowedActions['type'],
   AllowedEvents['type']
 >;
-const supportedNftDetectionNetworks: Hex[] = [ChainId.mainnet];
+const supportedNftDetectionNetworks: Hex[] = [
+  ChainId.mainnet,
```

Comment:
Do we have some tests that checks that we `detectNfts` only for theses two networks ? if not what do you think about adding some ?

Reply:
Thanks @cryptodev-2s!  i have added a unit test covering detecting NFTs on linea mainnet 🙏